apps/accounts/views.py

```diff
@@ -271,6 +279,9 @@ def done(self, form_list, form_dict, **kwargs):
         consent.request = pr
         consent.save()
 
+        # send an email notification to the author and green web staff
+        self._send_notification_email(self.request.user, pr)
```

Comment:
Since `_send_notification_email` is defined in the same class, there's no need to pass `self.request.user` as a parameter - the user object can be retrieved inside `_send_notification_email`, since it has access to `self`.